x-pack/metricbeat/module/azure/compute_vm/compute_vm_integration_test.go

```diff
@@ -2,9 +2,6 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
```

Comment:
integration headers are missing

Reply:
ahh sorry I removed them for testing and forgot to add them back 😄 Will do!

Reply:
done.